packages/google_sign_in/google_sign_in_web/test/auth2_test.dart

```diff
@@ -48,7 +48,7 @@ void main() {
 
   group('Successful .init, then', () {
     setUp(() async {
-      plugin.init(
+      await plugin.init(
```

Comment:
Was this causing trouble with the tests?

There's really no need to await the init, since all the plugin operations do this already. Maybe remove `async` from the setUp method if it was a lint issue?

Reply:
Yeah, the lint here was `unawaited_futures`. If the plugin itself is deliberately trying to support this case I think it makes sense to not await it in the test. I changed it back and removed the `async` to avoid catching the lint.